### ISSUE TITLE
Mangas-origines checking if site connection is secure

### ISSUE THREAD

**Issue by JetonBreton** [tachiyomiorg/tachiyomi-extensions#14411]
### Source information and language

Mangas Origines v1.3.29 (français)

### Steps to reproduce

open manga origins choose a reference and try to load a chapter even when switching to the web version

### Expected behavior

blocked by checking if site connection is secure on reading chapters but not on searching for recent and popular titles 

### Actual behavior

chapter loading

### Tachiyomi version

0.14.2

### Android version

Android 12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by alessandrojean** [comment]
Refer to the Solving Cloudflare issues section at https://tachiyomi.org/help/guides/troubleshooting/#solving-cloudflare-issues

If they don't work, migrate to other sources or wait until they lower their protection.
